mozilla/bedrock
# Update accounts page

Updates Firefox Accounts page to change focus to account features and away from family of products.

## Significant changes and points to review

- put sign in form first on page
- list products an account can link to with short list of benefits
- remove 'join firefox' campaign language
- remove unused assets

## Issue / Bugzilla link

n/a

## Testing

http://localhost:8000/en-US/firefox/accounts/

- check L10N closely, I deleted a bunch of strings and added some new ones
- check signed in and signed out of Firefox status & the form
